icaruscode/Decode/fcl/decoderdefs_icarus.fcl

```diff
@@ -55,4 +55,10 @@ decodeTrigger: {
                     DecoderTool:        @local::TriggerDecoderTool
 }
 
+decodeTriggerV2: {
+		   module_type:	        DaqDecoderICARUSTriggerV2
```

Comment:
I would have thought that the old module `DaqDecoderICARUSTrigger` chould serve both the old "legacy" tool and the "V2" tool.

Reply:
Okay I see, this is now removed and there is one module to handle both cases